src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -25,8 +25,8 @@
 {
     protected $name;
     protected $parent;
-    protected $normalizationClosures = array();
-    protected $finalValidationClosures = array();
+    protected $normalizationCallbacks = array();
+    protected $finalValidationCallbacks = array();
```

Comment:
Aliasing the old ones by reference would allow preserving BC.
you should do it in the constructor, and for completeness also add a magic __clone method:
```php

public function __construct(...)
{
    $this->normalizationClosures = &$this->normalizationCallbacks;
    $this->finalValidationClosures = &$this->finalValidationCallbacks;

    // ...
}
// ...
public function __clone()
{
    // Break cross-clones references but preserve them inside each one.
    unset($this->normalizationCallbacks, $this->finalValidationCallbacks);
    $this->normalizationCallbacks = $this->normalizationClosures;
    $this->finalValidationCallbacks = $this->finalValidationClosures;
    $this->normalizationClosures = &$this->normalizationCallbacks;
    $this->finalValidationClosures = &$this->finalValidationCallbacks;
}
```